ragelo/types/types.py

```diff
@@ -227,6 +206,44 @@ def add_retrieved_by(
             score = 1.0
         self.retrieved_by[agent] = score
 
+class ChatMessage(BaseModel):
+    sender: str
+    content: str
+    # referenced_docs: Optional[List[Document]] = None
```

Comment:
Do you plan on adding this in the future? Otherwise, just remove the comment

Reply:
right, I will remove it. Was thinking to have it first, but since the docs are in the Query we don't need. I guess it's similar to how the agent's `answer` works which might contain citation but the docs are still added at the Query level.